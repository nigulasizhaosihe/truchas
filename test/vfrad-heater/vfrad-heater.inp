&MESH
  mesh_file = "mesh.gen"
/

&OUTPUTS
  output_t = 0.0, 1.0
  output_dt  = 1.0
/

&PHYSICS
  heat_transport = T
  materials = "solid"
/

&PHYSICAL_CONSTANTS
  absolute_zero = 0.0
  stefan_boltzmann = 1.0e-2
/

&NUMERICS
  dt_init = 1e-3
  dt_min  = 1e-6
  dt_grow = 10.0
/

&DIFFUSION_SOLVER
  abs_enthalpy_tol          = 0.0
  rel_enthalpy_tol          = 0.001
  abs_temp_tol              = 0.0
  rel_temp_tol              = 0.001
  nlk_preconditioner        = "Hypre_AMG"
  max_nlk_itr               = 5
  max_nlk_vec               = 4
  nlk_tol                   = 0.05
  pc_amg_cycles             = 2
  verbose_stepping          = T
/

&MATERIAL
  name = "solid"
  is_fluid = .false.
  density = 4.0
  specific_heat = 0.25
  conductivity = 0.1
/

Heating element
&BODY
  surface_name = "from mesh file"
  mesh_material_number = 2
  material_name = "solid"
  temperature = 6.0
/

Container
&BODY
  surface_name = "from mesh file"
  mesh_material_number = 1
  material_name = "solid"
  temperature = 1.0
/

Thermal radiation inside the container

&ENCLOSURE_RADIATION
  name                      = "encl"
  enclosure_file            = "encl.re"
  ambient_constant          = 1.0
  error_tolerance           = 0.001
  precon_method             = "jacobi"
  precon_iter               = 1
  precon_coupling_method    = "backward GS"
/

&ENCLOSURE_SURFACE
  enclosure_name            = "encl"
  name                      = "cavity"
  face_block_ids            = 2, 3
  emissivity_constant       = 0.8
/

Surface temperature of the heating element is imposed
&THERMAL_BC
  name = "heater"
  type = "temperature"
  face_set_ids = 2
  temp = 6.0
/

Outside of container radiates to ambient
&THERMAL_BC
  name = "outer"
  type = "radiation"
  face_set_ids = 4
  emissivity = 0.8
  ambient_temp = 1.0
/

Symmetry planes
&THERMAL_BC
  name = "symmetry"
  type = "flux"
  face_set_ids = 1, 3
  flux = 0.0
/
