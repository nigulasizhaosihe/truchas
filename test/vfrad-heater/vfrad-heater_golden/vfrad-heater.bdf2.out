
TRAP step      1: T= 0.000000000000000E+00, H= 1.000000000000000E-03
  Preconditioner updated at T= 1.000000000000000E-03
    1: error= 7.232952028040196E+00
    2: error= 7.692079835732377E-02
    3: error= 1.022918305432545E-02
    4: error= 9.832446367997560E-04
  NLK BCE solve succeeded:   4 iterations, error= 9.832446367997560E-04
  Step accepted: no local error control

BDF2 step      2: T= 1.000000000000000E-03, H= 1.000000000000000E-03
    1: error= 1.714530841255216E+00
    2: error= 3.943525090809712E-01
    3: error= 4.277032147199886E-02
    4: error= 3.637880934630910E-03
  NLK BCE solve succeeded:   4 iterations, error= 3.637880934630910E-03
  Step accepted: no local error control

BDF2 step      3: T= 2.000000000000000E-03, H= 1.000000000000000E-03
    1: error= 1.589133128013955E+00
    2: error= 4.001083510628322E-01
    3: error= 6.530897765402247E-03
  NLK BCE solve succeeded:   3 iterations, error= 6.530897765402247E-03
  Step accepted: no local error control

BDF2 step      4: T= 3.000000000000000E-03, H= 1.000000000000000E-03
    1: error= 1.132893028207891E+00
    2: error= 2.876637397340189E-01
    3: error= 4.842220181370630E-03
  NLK BCE solve succeeded:   3 iterations, error= 4.842220181370630E-03
  Step accepted: perr= 1.414884886789685E+00

BDF2 step      5: T= 4.000000000000000E-03, H= 5.413151452770080E-04
    1: error= 3.675035615359846E-01
    2: error= 6.730449823441453E-02
    3: error= 1.474779753060188E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.474779753060188E-03
  Step accepted: perr= 3.004102465367224E-01

BDF2 step      6: T= 4.541315145277008E-03, H= 9.499281805999092E-04
    1: error= 5.701022262660725E-01
    2: error= 8.035998302968518E-02
    3: error= 7.321527856681559E-04
  NLK BCE solve succeeded:   3 iterations, error= 7.321527856681559E-04
  Step accepted: perr= 6.495112572055710E-01

BDF2 step      7: T= 5.491243325876917E-03, H= 7.288845388408712E-04
    1: error= 2.806277967905586E-01
    2: error= 5.573592089413612E-03
  NLK BCE solve succeeded:   2 iterations, error= 5.573592089413612E-03
  Step accepted: perr= 2.840890440423904E-01

BDF2 step      8: T= 6.220127864717788E-03, H= 1.016436488924148E-03
    1: error= 3.708528187896355E-01
    2: error= 9.169779217281133E-02
    3: error= 1.466968552422941E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.466968552422941E-03
  Step accepted: perr= 4.624250726201596E-01

BDF2 step      9: T= 7.236564353641937E-03, H= 9.637406665849125E-04
    1: error= 2.822257691702934E-01
    2: error= 7.260627600265308E-02
    3: error= 1.378119933015042E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.378119933015042E-03
  Step accepted: perr= 3.549797425718472E-01

BDF2 step     10: T= 8.200305020226849E-03, H= 1.123809636070070E-03
    1: error= 3.017001655244045E-01
    2: error= 1.180743032854840E-01
    3: error= 3.308020240063113E-03
  NLK BCE solve succeeded:   3 iterations, error= 3.308020240063113E-03
  Step accepted: perr= 4.218283236421232E-01

BDF2 step     11: T= 9.324114656296919E-03, H= 1.162020874794100E-03
    1: error= 2.653257418433669E-01
    2: error= 1.203958145061921E-01
    3: error= 4.680793611658660E-03
  NLK BCE solve succeeded:   3 iterations, error= 4.680793611658660E-03
  Step accepted: perr= 3.890925502495226E-01

BDF2 step     12: T= 1.048613553109102E-02, H= 1.276686802331981E-03
    1: error= 2.587898821371544E-01
    2: error= 1.436283316991711E-01
    3: error= 8.809639891887476E-03
  NLK BCE solve succeeded:   3 iterations, error= 8.809639891887476E-03
  Step accepted: perr= 4.098947111570976E-01

BDF2 step     13: T= 1.176282233342300E-02, H= 1.355102377722116E-03
    1: error= 2.386626937284633E-01
    2: error= 1.502053491268732E-01
    3: error= 1.322005857097853E-02
    4: error= 1.495518827259854E-03
  NLK BCE solve succeeded:   4 iterations, error= 1.495518827259854E-03
  Step accepted: perr= 4.006740726754927E-01

BDF2 step     14: T= 1.311792471114512E-02, H= 1.463080630507515E-03
    1: error= 2.252731382768355E-01
    2: error= 1.590256484624213E-01
    3: error= 2.094389927120470E-02
    4: error= 1.957163619107061E-03
  NLK BCE solve succeeded:   4 iterations, error= 1.957163619107061E-03
  Step accepted: perr= 4.037457582972054E-01

BDF2 step     15: T= 1.458100534165263E-02, H= 1.570522918275065E-03
    1: error= 2.092717669915293E-01
    2: error= 1.633641698520203E-01
    3: error= 2.902856318134398E-02
    4: error= 2.206914942674380E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.206914942674380E-03
  Step accepted: perr= 3.998496305385772E-01

BDF2 step     16: T= 1.615152825992770E-02, H= 1.694080435597607E-03
    1: error= 1.953121649822145E-01
    2: error= 1.649248897703810E-01
    3: error= 4.058100549699323E-02
    4: error= 2.585487159871324E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.585487159871324E-03
  Step accepted: perr= 3.988958489583095E-01

BDF2 step     17: T= 1.784560869552530E-02, H= 1.827569844087197E-03
    1: error= 1.815523685959149E-01
    2: error= 1.632451342409017E-01
    3: error= 5.376458650444865E-02
    4: error= 3.014601048066932E-03
  NLK BCE solve succeeded:   4 iterations, error= 3.014601048066932E-03
  Step accepted: perr= 3.966323226979829E-01

BDF2 step     18: T= 1.967317853961250E-02, H= 1.975878518175645E-03
    1: error= 1.687419951017856E-01
    2: error= 1.591122284327744E-01
    3: error= 7.171857243161020E-02
    4: error= 3.710673686946581E-03
  NLK BCE solve succeeded:   4 iterations, error= 3.710673686946581E-03
  Step accepted: perr= 3.948496799626734E-01

BDF2 step     19: T= 2.164905705778815E-02, H= 2.139288939347894E-03
    1: error= 1.574921032028328E-01
    2: error= 1.598118335046120E-01
    3: error= 9.412816046487733E-02
    4: error= 4.963209643036649E-03
  NLK BCE solve succeeded:   4 iterations, error= 4.963209643036649E-03
  Step accepted: perr= 4.156658159005006E-01

BDF2 step     20: T= 2.378834599713604E-02, H= 2.251340306018475E-03
    1: error= 1.500640832585741E-01
    2: error= 1.537096405445055E-01
    3: error= 1.055068799052581E-01
    4: error= 6.211182967920529E-03
  NLK BCE solve succeeded:   4 iterations, error= 6.211182967920529E-03
  Step accepted: perr= 4.149342697875043E-01

BDF2 step     21: T= 2.603968630315451E-02, H= 2.400474518242188E-03
    1: error= 1.417121890987650E-01
    2: error= 1.466574707574665E-01
    3: error= 1.160695270655239E-01
    4: error= 8.435176186161068E-03
  NLK BCE solve succeeded:   4 iterations, error= 8.435176186161068E-03
  Step accepted: perr= 4.122847452419641E-01

BDF2 step     22: T= 2.844016082139670E-02, H= 2.565753921569533E-03
    1: error= 1.308794826235632E-01
    2: error= 1.354535090657261E-01
    3: error= 1.239789067012452E-01
    4: error= 1.255923830750329E-02
    5: error= 1.399443069963345E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.399443069963345E-03
  Step accepted: perr= 4.031390949959197E-01

BDF2 step     23: T= 3.100591474296624E-02, H= 2.766409894932997E-03
    1: error= 1.205758629437043E-01
    2: error= 1.220492324208082E-01
    3: error= 1.321247661844348E-01
    4: error= 2.035503705624690E-02
    5: error= 1.826297602825568E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.826297602825568E-03
  Step accepted: perr= 3.962133376008575E-01

BDF2 step     24: T= 3.377232463789923E-02, H= 2.997811263277507E-03
    1: error= 1.185117176050588E-01
    2: error= 1.107064975853848E-01
    3: error= 1.325571990167031E-01
    4: error= 3.414996830657580E-02
    5: error= 2.707462342432632E-03
  NLK BCE solve succeeded:   5 iterations, error= 2.707462342432632E-03
  Step accepted: perr= 3.906094135548016E-01

BDF2 step     25: T= 3.677013590117674E-02, H= 3.261460422518146E-03
  Preconditioner updated at T= 4.003159632369489E-02
    1: error= 3.823763987753986E-01
    2: error= 6.466142176478614E-03
  NLK BCE solve succeeded:   2 iterations, error= 6.466142176478614E-03
  Step accepted: perr= 3.852263334365706E-01

BDF2 step     26: T= 4.003159632369489E-02, H= 3.565498553646643E-03
    1: error= 3.463370958762920E-01
    2: error= 3.202995064076312E-02
    3: error= 1.137961569687349E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.137961569687349E-03
  Step accepted: perr= 3.764427226769942E-01

BDF2 step     27: T= 4.359709487734153E-02, H= 3.935071222662276E-03
    1: error= 3.155329790615348E-01
    2: error= 5.675268946029632E-02
    3: error= 1.603167912590944E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.603167912590944E-03
  Step accepted: perr= 3.713254192568392E-01

BDF2 step     28: T= 4.753216610000380E-02, H= 4.353344708022552E-03
    1: error= 2.860659154145381E-01
    2: error= 7.765050432806195E-02
    3: error= 3.455008261378641E-03
  NLK BCE solve succeeded:   3 iterations, error= 3.455008261378641E-03
  Step accepted: perr= 3.651172567682894E-01

BDF2 step     29: T= 5.188551080802636E-02, H= 4.846841527029813E-03
    1: error= 2.562327656412489E-01
    2: error= 9.517945938934189E-02
    3: error= 8.586663422932588E-03
  NLK BCE solve succeeded:   3 iterations, error= 8.586663422932588E-03
  Step accepted: perr= 3.583251051115223E-01

BDF2 step     30: T= 5.673235233505617E-02, H= 5.433143366455805E-03
    1: error= 2.293820549699224E-01
    2: error= 1.049571470306914E-01
    3: error= 1.898207199301132E-02
    4: error= 2.657179892827679E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.657179892827679E-03
  Step accepted: perr= 3.539437465201390E-01

BDF2 step     31: T= 6.216549570151197E-02, H= 6.106145183294748E-03
    1: error= 2.109924847155754E-01
    2: error= 1.141375526728168E-01
    3: error= 4.030442173266271E-02
    4: error= 6.125939173538224E-03
  NLK BCE solve succeeded:   4 iterations, error= 6.125939173538224E-03
  Step accepted: perr= 3.653996208339882E-01

BDF2 step     32: T= 6.827164088480672E-02, H= 6.733710947735477E-03
    1: error= 2.340002523945395E-01
    2: error= 1.364341224250323E-01
    3: error= 6.471386983932227E-02
    4: error= 1.258485677716157E-02
    5: error= 1.477498522111507E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.477498522111507E-03
  Step accepted: perr= 4.390383807166188E-01

BDF2 step     33: T= 7.500535183254220E-02, H= 6.783312148107165E-03
    1: error= 2.353687925761023E-01
    2: error= 1.364202202527799E-01
    3: error= 6.331367885531694E-02
    4: error= 1.592078293333444E-02
    5: error= 2.028164340415604E-03
  NLK BCE solve succeeded:   5 iterations, error= 2.028164340415604E-03
  Step accepted: perr= 4.360822635779867E-01

BDF2 step     34: T= 8.178866398064936E-02, H= 7.157576530278639E-03
    1: error= 2.937783904249346E-01
    2: error= 1.503943908793067E-01
    3: error= 6.667258069364816E-02
    4: error= 2.180595139915578E-02
    5: error= 3.102031700260069E-03
  NLK BCE solve succeeded:   5 iterations, error= 3.102031700260069E-03
  Step accepted: perr= 4.658042164487590E-01

BDF2 step     35: T= 8.894624051092800E-02, H= 7.251647032622585E-03
    1: error= 3.144318348418225E-01
    2: error= 1.541213122226254E-01
    3: error= 5.951251682532414E-02
    4: error= 2.471901595568443E-02
    5: error= 3.938575553784351E-03
  NLK BCE solve succeeded:   5 iterations, error= 3.938575553784351E-03
  Step accepted: perr= 4.587840067435134E-01

BDF2 step     36: T= 9.619788754355059E-02, H= 7.482223751412845E-03
    1: error= 3.236678322869692E-01
    2: error= 1.634983844334153E-01
    3: error= 5.627126500040952E-02
    4: error= 2.844864909015193E-02
    5: error= 5.107686152194832E-03
  NLK BCE solve succeeded:   5 iterations, error= 5.107686152194832E-03
  Step accepted: perr= 4.724926601325979E-01

BDF2 step     37: T= 1.036801112949634E-01, H= 7.590079233132430E-03
    1: error= 3.106393629971237E-01
    2: error= 1.653748695848894E-01
    3: error= 5.185424251149681E-02
    4: error= 2.933182720321778E-02
    5: error= 5.853616012851536E-03
  NLK BCE solve succeeded:   5 iterations, error= 5.853616012851536E-03
  Step accepted: perr= 4.649370493854540E-01

BDF2 step     38: T= 1.112701905280959E-01, H= 7.801239680436167E-03
    1: error= 2.943400853377078E-01
    2: error= 1.675787327205709E-01
    3: error= 5.120767665531810E-02
    4: error= 3.106947695082854E-02
    5: error= 6.740997062651568E-03
  NLK BCE solve succeeded:   5 iterations, error= 6.740997062651568E-03
  Step accepted: perr= 4.551871183990282E-01

BDF2 step     39: T= 1.190714302085320E-01, H= 8.087230442585330E-03
    1: error= 2.762507317407312E-01
    2: error= 1.671765981555759E-01
    3: error= 5.304328011610120E-02
    4: error= 3.419629088268709E-02
    5: error= 7.818050725385394E-03
  NLK BCE solve succeeded:   5 iterations, error= 7.818050725385394E-03
  Step accepted: perr= 4.546682519181658E-01

BDF2 step     40: T= 1.271586606511174E-01, H= 8.337638676919779E-03
    1: error= 2.542263498482539E-01
    2: error= 1.588395143123607E-01
    3: error= 5.395691899027504E-02
    4: error= 3.651093238406407E-02
    5: error= 8.519267388505732E-03
  NLK BCE solve succeeded:   5 iterations, error= 8.519267388505732E-03
  Step accepted: perr= 4.386426673674960E-01

BDF2 step     41: T= 1.354962993280371E-01, H= 8.772018893199507E-03
    1: error= 2.388165155493804E-01
    2: error= 1.494879991606285E-01
    3: error= 5.581181207768611E-02
    4: error= 4.166797064361894E-02
    5: error= 9.465964910450530E-03
  NLK BCE solve succeeded:   5 iterations, error= 9.465964910450530E-03
  Step accepted: perr= 4.242520144007501E-01

BDF2 step     42: T= 1.442683182212366E-01, H= 9.320452606320861E-03
    1: error= 2.275517262266187E-01
    2: error= 1.407274459866620E-01
    3: error= 5.658679812838661E-02
    4: error= 4.837197171880044E-02
    5: error= 1.029484472036618E-02
  NLK BCE solve FAILED:   5 iterations (max), error= 1.029484472036618E-02
  Preconditioner updated at T= 1.535887708275575E-01
    1: error= 4.128013524828405E-01
    2: error= 5.492610878391047E-02
    3: error= 2.267077855384139E-02
    4: error= 2.346958176075428E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.346958176075428E-03
  Step accepted: perr= 4.190417179857254E-01

BDF2 step     43: T= 1.535887708275575E-01, H= 9.891543936246944E-03
    1: error= 3.962645812868124E-01
    2: error= 6.289484943778116E-02
    3: error= 1.850464683885290E-02
    4: error= 2.954665985900788E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.954665985900788E-03
  Step accepted: perr= 4.201811598211079E-01

BDF2 step     44: T= 1.634803147638045E-01, H= 1.047038155044142E-02
    1: error= 3.745218431065875E-01
    2: error= 6.510825500271211E-02
    3: error= 1.123921959731563E-02
    4: error= 2.577817195613044E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.577817195613044E-03
  Step accepted: perr= 4.147493508125304E-01

BDF2 step     45: T= 1.739506963142459E-01, H= 1.117544299850054E-02
    1: error= 3.606989785756597E-01
    2: error= 6.423949053584159E-02
    3: error= 4.177346959535663E-03
  NLK BCE solve succeeded:   3 iterations, error= 4.177346959535663E-03
  Step accepted: perr= 4.175611985826625E-01

BDF2 step     46: T= 1.851261393127464E-01, H= 1.184703845409688E-02
    1: error= 3.445362366348411E-01
    2: error= 7.869569414071269E-02
    3: error= 5.103511747197365E-03
  NLK BCE solve succeeded:   3 iterations, error= 5.103511747197365E-03
  Step accepted: perr= 4.170435916508683E-01

BDF2 step     47: T= 1.969731777668433E-01, H= 1.258730431950175E-02
    1: error= 3.411563554763772E-01
    2: error= 9.356755706795612E-02
    3: error= 1.449738315776999E-02
    4: error= 8.690078083936353E-03
  NLK BCE solve succeeded:   4 iterations, error= 8.690078083936353E-03
  Step accepted: perr= 4.154280001865479E-01

BDF2 step     48: T= 2.095604820863450E-01, H= 1.340370941275060E-02
    1: error= 3.528806739964140E-01
    2: error= 1.072927173261192E-01
    3: error= 2.397946323559673E-02
    4: error= 1.273432415089402E-02
    5: error= 1.228657813249780E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.228657813249780E-03
  Step accepted: perr= 4.320328188636030E-01

BDF2 step     49: T= 2.229641914990956E-01, H= 1.395727664336316E-02
    1: error= 4.062707642583191E-01
    2: error= 1.229120435432205E-01
    3: error= 3.556190893884825E-02
    4: error= 1.331985132152817E-02
    5: error= 1.175982083149784E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.175982083149784E-03
  Step accepted: perr= 4.996505569109815E-01

BDF2 step     50: T= 2.369214681424588E-01, H= 1.355868409790983E-02
    1: error= 4.096486631873844E-01
    2: error= 1.272645129617503E-01
    3: error= 3.630557125888750E-02
    4: error= 8.946957058705280E-03
  NLK BCE solve succeeded:   4 iterations, error= 8.946957058705280E-03
  Step accepted: perr= 4.998487657885952E-01

BDF2 step     51: T= 2.504801522403686E-01, H= 1.364121184657252E-02
    1: error= 3.940786267877982E-01
    2: error= 1.322462965858085E-01
    3: error= 3.938044918611232E-02
    4: error= 5.318063523484495E-03
  NLK BCE solve succeeded:   4 iterations, error= 5.318063523484495E-03
  Step accepted: perr= 4.830526792738775E-01

BDF2 step     52: T= 2.641213640869411E-01, H= 1.393481350307690E-02
    1: error= 3.653593342284090E-01
    2: error= 1.370394633918708E-01
    3: error= 4.001603280938783E-02
    4: error= 5.054191623709458E-03
  NLK BCE solve succeeded:   4 iterations, error= 5.054191623709458E-03
  Step accepted: perr= 4.624961213424190E-01

BDF2 step     53: T= 2.780561775900180E-01, H= 1.438244023484458E-02
    1: error= 3.346041146058011E-01
    2: error= 1.420445452894243E-01
    3: error= 4.053975967897266E-02
    4: error= 5.869863032835812E-03
  NLK BCE solve succeeded:   4 iterations, error= 5.869863032835812E-03
  Step accepted: perr= 4.449315977500996E-01

BDF2 step     54: T= 2.924386178248626E-01, H= 1.504730323275794E-02
    1: error= 3.061195285701269E-01
    2: error= 1.498583820119884E-01
    3: error= 4.116749824383076E-02
    4: error= 6.634149254271447E-03
  NLK BCE solve succeeded:   4 iterations, error= 6.634149254271447E-03
  Step accepted: perr= 4.354363824495686E-01

BDF2 step     55: T= 3.074859210576206E-01, H= 1.580248100503873E-02
    1: error= 2.782193429240671E-01
    2: error= 1.562893030556359E-01
    3: error= 4.114070589837266E-02
    4: error= 7.193960910595058E-03
  NLK BCE solve succeeded:   4 iterations, error= 7.193960910595058E-03
  Step accepted: perr= 4.207337115892610E-01

BDF2 step     56: T= 3.232884020626593E-01, H= 1.683662773345818E-02
    1: error= 2.541052966917786E-01
    2: error= 1.661404600332833E-01
    3: error= 4.159513622746729E-02
    4: error= 7.849896217664680E-03
  NLK BCE solve succeeded:   4 iterations, error= 7.849896217664680E-03
  Step accepted: perr= 4.108721232221388E-01

BDF2 step     57: T= 3.401250297961175E-01, H= 1.804113954577696E-02
    1: error= 2.299140197424286E-01
    2: error= 1.749995306187631E-01
    3: error= 4.159366841347585E-02
    4: error= 8.433343625137835E-03
  NLK BCE solve succeeded:   4 iterations, error= 8.433343625137835E-03
  Step accepted: perr= 3.988528360366815E-01

BDF2 step     58: T= 3.581661693418944E-01, H= 1.954415406478582E-02
    1: error= 2.076895424526770E-01
    2: error= 1.847934395792820E-01
    3: error= 4.165745420209217E-02
    4: error= 9.056813612456338E-03
  NLK BCE solve succeeded:   4 iterations, error= 9.056813612456338E-03
  Step accepted: perr= 3.898947513780018E-01

BDF2 step     59: T= 3.777103234066803E-01, H= 2.130820367328862E-02
    1: error= 1.872724126563824E-01
    2: error= 1.926030229834240E-01
    3: error= 4.133781314894909E-02
    4: error= 9.590602626737162E-03
  NLK BCE solve succeeded:   4 iterations, error= 9.590602626737162E-03
  Step accepted: perr= 3.788812428679993E-01

BDF2 step     60: T= 3.990185270799689E-01, H= 2.348020044379129E-02
    1: error= 1.674891186502262E-01
    2: error= 1.995449959241162E-01
    3: error= 4.101477284891677E-02
    4: error= 1.011979473146933E-02
    5: error= 1.962047190229542E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.962047190229542E-03
  Step accepted: perr= 3.662356113135564E-01

BDF2 step     61: T= 4.224987275237602E-01, H= 2.619477065736797E-02
    1: error= 1.508000408795183E-01
    2: error= 2.080023958944605E-01
    3: error= 4.108658720274011E-02
    4: error= 1.104995959929707E-02
    5: error= 1.988278032682267E-03
  NLK BCE solve succeeded:   5 iterations, error= 1.988278032682267E-03
  Step accepted: perr= 3.596740000870340E-01

BDF2 step     62: T= 4.486934981811281E-01, H= 2.929855925113978E-02
    1: error= 1.293500755627409E-01
    2: error= 2.050599010832122E-01
    3: error= 4.118860907160012E-02
    4: error= 9.920121856360034E-03
  NLK BCE solve succeeded:   4 iterations, error= 9.920121856360034E-03
  Step accepted: perr= 3.409838169482446E-01

BDF2 step     63: T= 4.779920574322679E-01, H= 3.359917729682282E-02
    1: error= 1.073770123841751E-01
    2: error= 2.003540482339913E-01
    3: error= 3.979051159495230E-02
    4: error= 9.787447732723114E-03
  NLK BCE solve succeeded:   4 iterations, error= 9.787447732723114E-03
  Step accepted: perr= 3.194063105118257E-01

BDF2 step     64: T= 5.115912347290907E-01, H= 3.943541439430276E-02
  Preconditioner updated at T= 5.510266491233935E-01
    1: error= 2.815892344159099E-01
    2: error= 4.690423922363320E-02
    3: error= 4.371358297736850E-02
    4: error= 8.546195860188887E-04
  NLK BCE solve succeeded:   4 iterations, error= 8.546195860188887E-04
  Step accepted: perr= 3.065224135829846E-01

BDF2 step     65: T= 5.510266491233935E-01, H= 4.665665520416373E-02
    1: error= 2.420030828237392E-01
    2: error= 5.331565922527505E-02
    3: error= 4.064462669648734E-02
    4: error= 9.642655073436349E-04
  NLK BCE solve succeeded:   4 iterations, error= 9.642655073436349E-04
  Step accepted: perr= 2.919608453761921E-01

BDF2 step     66: T= 5.976833043275572E-01, H= 5.622901159769111E-02
    1: error= 1.817718307487052E-01
    2: error= 6.315992978890692E-02
    3: error= 3.797329416218091E-02
    4: error= 1.202434582307570E-03
  NLK BCE solve succeeded:   4 iterations, error= 1.202434582307570E-03
  Step accepted: perr= 2.501343959348098E-01

BDF2 step     67: T= 6.539123159252483E-01, H= 7.151409232130601E-02
    1: error= 1.352117324455929E-01
    2: error= 7.405562235967905E-02
    3: error= 3.450941823863805E-02
    4: error= 2.018126907177118E-03
  NLK BCE solve succeeded:   4 iterations, error= 2.018126907177118E-03
  Step accepted: perr= 2.157905575356640E-01

BDF2 step     68: T= 7.254264082465544E-01, H= 7.034692024945788E-02
    1: error= 6.130904307671396E-02
    2: error= 3.494656779648512E-02
    3: error= 1.609615507016147E-02
    4: error= 9.606130615750948E-04
  NLK BCE solve succeeded:   4 iterations, error= 9.606130615750948E-04
  Step accepted: perr= 9.949426562558296E-02

BDF2 step     69: T= 7.957733284960122E-01, H= 6.919879743910606E-02
    1: error= 2.512165160662755E-02
    2: error= 1.393462397336120E-02
    3: error= 6.484539735227366E-03
  NLK BCE solve succeeded:   3 iterations, error= 6.484539735227366E-03
  Step accepted: perr= 4.058853271874638E-02

BDF2 step     70: T= 8.649721259351183E-01, H= 6.806941298976543E-02
    1: error= 7.235896405740852E-03
    2: error= 3.986061041874624E-03
  NLK BCE solve succeeded:   2 iterations, error= 3.986061041874624E-03
  Step accepted: perr= 1.118321641521384E-02

BDF2 step     71: T= 9.330415389248837E-01, H= 6.695846107511627E-02
    1: error= 2.775810576803899E-03
    2: error= 4.004977152656753E-03
  NLK BCE solve succeeded:   2 iterations, error= 4.004977152656753E-03
  Step accepted: perr= 6.725431940122784E-03
